packages/react-menu/src/components/MenuTrigger/useTriggerElement.ts

```diff
@@ -81,10 +81,6 @@ export const useTriggerElement = (state: MenuTriggerState): MenuTriggerState =>
       focusFirst();
     }
 
-    if (open && key === ArrowDown && !isSubmenu) {
```

Comment:
Now this case should never happen, (menu is open and focus is still on the trigger)